cpp/src/ray/util/function_helper.cc

```diff
@@ -86,6 +86,12 @@ std::string FunctionHelper::LoadAllRemoteFunctions(const std::string lib_path,
                      << "' not found in " << lib_path;
     return "";
   }
+  // In some compiler, the static ray runtime maybe a new instance in dynamic library, so
+  // we need init it in dynamic library to make sure the new instance valid.
```

Comment:
Can we add a test for this case?

Reply:
It's hard to test, the ci machines and ours machines work well, but in @MissiontoMars machine it crashed because of null ray runtime, i have tested in MissiontoMars machine, it fixed.